v1.1/initialize-a-cluster.md

```diff
@@ -4,7 +4,7 @@ summary: Perform a one-time-only initialization of a CockroachDB cluster.
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> This page explains the `cockroach init` [command](cockroach-commands.html), which you use to perform a one-time initialization of a new multi-node cluster. For a full walk-through of the cluster startup and initialization process, see one of the [Manual Deployment](manual-deployment.html) tutorials.
+<span class="version-tag">New in v1.1:</span> This page explains the `cockroach init` [command](cockroach-commands.html), which you use to perform a one-time initialization of a new multi-node cluster. The target of the `cockroach init` command must appear in the `--join` flag of the [`cockroach start`](start-a-node.html) command which have been run on the other nodes in the cluster. For a full walk-through of the cluster startup and initialization process, see one of the [Manual Deployment](manual-deployment.html) tutorials.
```

Comment:
Ugh, I seem to have not submitted this comment before. Sorry.

Instead of mentioning this here, I'd suggest updating the `--host` flag description as follows:

> The server host to connect to. This must appear in the `--join` flag of other nodes. 

@bdarnell, does the target host address need to appear in the join flag of all or any of the other nodes?

Reply:
It's `--advertise-host`, not `--host` that must appear in `--join` and be used from client commands.

We recommend using the same `--join` arguments on all nodes, and while they may diverge over time they should at least be the same at `init` time. 

Putting this message in the first paragraph of the command seems strange to me; I would put it in the docs for the `--host` flag. I don't think we need to make this warning too prominent since at the time that you're initializing the cluster you shouldn't have too many nodes or a complex configuration.

Reply:
I'm missing something, @bdarnell. `--advertise-host` isn't even an option on client commands like `init`...  Or are you just saying that, when starting a cluster, if you use `--advertise-host` on nodes, that's what needs to be used in `--join` as well?

Reply:
Sorry, I got mixed up. I thought you were suggesting that the server-side `--host` flag should say that this is what should appear in the `--join` flag.

In retrospect, instead of `--advertise-host` and `--host`, we should have had `--host` and `--bind-host`. It's the advertised host that matters everywhere but on the node itself.